src/agent/rustjail/src/mount.rs

```diff
@@ -288,6 +289,35 @@ pub fn init_rootfs(
         }
     }
 
+    if expose_configfs {
+        if let Some(annotations) = spec.annotations() {
+            annotations.iter().for_each(|(k, v)| {
+                if k == "io.katacontainers.pkg.oci.container_type" && v != "pod_sandbox" {
```

Comment:
IMHO, What's about using [the public const value ](https://github.com/kata-containers/kata-containers/blob/d7059e9024d456bb81aaa974eff0bd9e4cc1d977/src/libs/kata-types/src/annotations/mod.rs#L48) to replace the `"io.katacontainers.pkg.oci.container_type"` ?

Reply:
@Apokleos, this will add a kata-sys-types dependency to the rustjail.

I have no problem doing it, but I also think that this could be done in a different PR and then checking / replacing all the constants being used.

Reply:
>
> but I also think that this could be done in a different PR and then checking / replacing all the constants being used.
>
Sure, @fidencio Agree.